v2/internal/binding/binding_test/binding_notags_test.go

```diff
@@ -5,6 +5,7 @@ type NoFieldTags struct {
 	Address string
 	Zip     *string
 	Spouse  *NoFieldTags
+	NoFunc  func() string
```

Comment:
Is this needed?

Reply:
Before this PR, this line would cause the same problem as the issue in 3809. So if it gets reverted then this will trigger the bug again.

Reply:
So not only should it not show up in the test output ~ line 28 of this file, but it would also cause a binding failure.